elastic/datadog_checks/elastic/metrics.py

```diff
@@ -258,6 +258,22 @@
     ),
 }
 
+JVM_METRICS_RATE = {
+    # Submit metrics as rate
+    'jvm.gc.collectors.young.rate': ('rate', 'jvm.gc.collectors.young.collection_count'),
+    'jvm.gc.collectors.young.collection_time.rate': (
+        'rate',
+        'jvm.gc.collectors.young.collection_time_in_millis',
+        lambda ms: ms_to_second(ms),
+    ),
+    'jvm.gc.collectors.old.rate': ('rate', 'jvm.gc.collectors.old.collection_count'),
+    'jvm.gc.collectors.old.collection_time.rate': (
+        'rate',
+        'jvm.gc.collectors.old.collection_time_in_millis',
```

Comment:
Would it make sense to consider making this a [distribution](https://docs.datadoghq.com/developers/metrics/types/?tab=distribution) ?

Apologies for the intrusion, btw. I ask because I worked on a patch to collect some additional index-specific metrics and was puzzled by the fact that there are no `count` or `distribution` metrics.